eng/build.ps1

```diff
@@ -0,0 +1,34 @@
+[CmdletBinding(PositionalBinding=$false)]
+Param(
+  [string] $ArchGroup,
+  [switch] $Release,
+  [switch] $c,
+  [switch] $Restore,
+  [switch] $Test,
+  [switch] $Pack,
+  [switch] $Sign,
+  [switch] $Publish,
+  [Parameter(ValueFromRemainingArguments=$true)][String[]]$ExtraArgs
+)
+
+$defaultargs = "-build -restore -warnaserror:0"
+
+foreach ($argument in $PSBoundParameters.Keys)
+{
+  switch($argument)
+  {
+    "Debug"        { $arguments += "-configuration Debug " }
+    "Release"      { $arguments += "-configuration Release " }
+    "ExtraArgs"    { $arguments += $ExtraArgs }
+    "c"            { $arguments += "-rebuild " }
+    "Publish"      { $arguments += "" }
```

Comment:
What is publish doing here?

Reply:
It's swallowing the `Publish` arg so it doesn't get passed through to Arcade's `build.proj`, to ensure we don't accidentally opt-in to Arcade's publishing before we're ready. For now, the pipeline invokes msbuild.ps1 on publish.msbuild instead.